### DISCUSSION TITLE
Allow null conditional be left side

### DISCUSSION THREAD

**Discussion by ahdung** [dotnet/csharplang#4463] [Language Ideas]
I hope can do this:
```c#
obj?.Prop = xxx; // if obj == null, do nothing
```

**Comment by jnm2** [comment]
Me too. This is currently championed at https://github.com/dotnet/csharplang/issues/2883.

**Reply by bernd5** [reply]
Thanks - and nice idea

**Reply by ahdung** [reply]
@jnm2 my fault

**Reply by jnm2** [reply]
@ahdung Don't worry, it can be hard to know how to find existing issues.
